src/js/ZoomPane.js

```diff
@@ -94,31 +115,21 @@ export default class ZoomPane {
       let scrollY = window.pageYOffset;
 
       let inlineLeft =
-        triggerRect.left +
```

Comment:
These changes are just performance changes, removing calls to `this.el.clientWidth` (or similar) and replacing them with `elWidth`, which reduces the number of DOM calls, improving performance